src/test/resources/integration-test-mule-config.xml

```diff
@@ -19,4 +19,14 @@
     <flow name="getCertificateTest" >
         <akv:get-certificate config-ref="config" certificateName="${akv.test.certificate.name}"/>
     </flow>
+
+    <flow name="encryptKeyTest" >
+        <akv:encrypt-key config-ref="config" keyName="${akv.test.key.name}" alg="${akv.test.alg}" value="${akv.test.value}" />
+    </flow>
+
+    <flow name="decryptKeyTest" >
+        <akv:encrypt-key config-ref="config" keyName="${akv.test.key.name}" alg="${akv.test.alg}" value="${akv.test.value}" />
+        <set-variable value="#[payload.value]" variableName="encryptedValue"/>
```

Comment:
don't really need set-variable, if you are just going to use payload.value in the 'decrypt-key' element...